src/index.ts

```diff
@@ -189,16 +188,16 @@ const sidebar: JupyterFrontEndPlugin<IDebuggerSidebar> = {
  */
 const tracker: JupyterFrontEndPlugin<IDebugger> = {
   id: '@jupyterlab/debugger:tracker',
-  optional: [ILayoutRestorer, IDebuggerSidebar, ICommandPalette],
-  requires: [IStateDB],
+  optional: [IDebuggerSidebar],
+  requires: [IStateDB, ICommandPalette, ILayoutRestorer],
```

Comment:
Any reason to move them to `requires`?

Reply:
We want to be able to restore the layout and we want the commands to be visible in the command palette, so I don't see any reason for having them as optional (but maybe there is some that I am not aware of).

Reply:
There could be custom builds of JupyterLab that don't provide these tokens (for example no command palette).

Reply:
ok, I'll move them back to optional then.